plugin/trino-druid/src/main/java/io/trino/plugin/druid/DruidJdbcClient.java

```diff
@@ -136,6 +144,68 @@ protected ResultSet getTables(Connection connection, Optional<String> schemaName
                 null);
     }
 
+    @Override
+    protected String toRemoteSchemaName(JdbcIdentity identity, Connection connection, String schemaName)
```

Comment:
Add a TODO comment here and `toRemoteTableName`.
Mention what has been changed from the original method in the comment so that it's easier to make sure code is equivalent when this override can be removed.